my-app/pom.xml

```diff
@@ -43,6 +43,16 @@
       <artifactId>vallang</artifactId>
       <version>0.14.4</version>
     </dependency>
+    <dependency>
+      <groupId>org.rascalmpl</groupId>
+      <artifactId>rascal</artifactId>
+      <version>0.23.0</version>
+    </dependency>
```

Comment:
Rascal dependency is added to be able to transform Java exceptions into Rascal exceptions